swarm/storage/localstore.go

```diff
@@ -74,4 +74,6 @@ func (self *LocalStore) Get(key Key) (chunk *Chunk, err error) {
 }
 
 // Close local store
-func (self *LocalStore) Close() {}
+func (self *LocalStore) Close() {
+	self.DbStore.Close()
+}
```

Comment:
Is this a bug fix or related to this PR?

Reply:
It belongs to `dbstore-fix` actually. But I put it in here since I can't base on that branch for now, and I need the db not to corrupt between sessions for tests .. :)